### DISCUSSION TITLE
Welcome to ProjectManager Discussions!

### DISCUSSION THREAD

**Discussion by NicolasPetton** [NicolasPetton/Projectify#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ludwa6** [comment]
I'm Walt -a relative n00b to TiddlyWiki and especially Projectify (which i only discovered yesterday) who's having a hard time resisting the temptation to fiddle/diddle with this technology, but determined to do so... Because i really need to be productive with it!

So: i'm probably going to bore everyone in this community stiff with [such naive questions](https://github.com/NicolasPetton/Projectify/discussions/22), but, well... That's all i got, for now!

**Reply by NicolasPetton** [reply]
Welcome! :wave:
